stream-chat-swift-docs/docs/guides/integration.md

```diff
@@ -2,8 +2,43 @@
 title: Integration
 ---
 
-## Cocoapods
+To get started integrating Stream Chat in your iOS app, install the `StreamChatUI` dependency using one of the following dependency managers.
 
-## SPM
+## CocoaPods
 
-## Carthage
\ No newline at end of file
+In your project's Podfile, add: `pod 'StreamChatUI', '~> 3.1'`. It should look similar to the snippet below.
+
+```ruby
+target 'ChatDemo' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for ChatDemo
+  pod 'StreamChatUI', '~> 3.1'
+end
+```
+
+Now that we’ve modified our Podfile, let’s go ahead and install the project dependencies via the terminal with one simple command:
+
+```bash
+pod install --repo-update
+```
+
+The above command will generate the **ChatDemo.xcworkspace** file automatically.
+
+With our workspace now containing our Pods project with dependencies, as well as our original project, let’s go ahead and move over to Xcode to complete the process.
+
+## Swift Package Manager
+
+Add the following to your Package.swift or in Xcode -> File -> Swift Packages -> Add Package Dependency:
+
+```swift
+dependencies: [
+    .package(url: "https://github.com/GetStream/stream-chat-swift.git", .upToNextMajor(from: "3.1"))
+]
```

Comment:
If Xcode UI is used, the code snippet is unnecessary, it'd be nice if you show the Xcode UI